doc/preface.chapter.md

```diff
@@ -6,11 +6,14 @@ The Nix Packages collection (Nixpkgs) is a set of thousands of packages for the
 Packages are available for several platforms, and can be used with the Nix
 package manager on most GNU/Linux distributions as well as [NixOS](https://nixos.org/nixos).
 
-This manual primarily describes how to write packages for the Nix Packages collection
-(Nixpkgs). Thus it’s mainly for packagers and developers who want to add packages to
-Nixpkgs. If you like to learn more about the Nix package manager and the Nix
-expression language, then you are kindly referred to the [Nix manual](https://nixos.org/nix/manual/).
-The NixOS distribution is documented in the [NixOS manual](https://nixos.org/nixos/manual/).
+This document is the user [_reference_](https://nix.dev/contributing/documentation/diataxis#reference) manual for Nixpkgs.
+Its purpose is to contain consise and complete listings of all functions, arguments and attributes with examples and cross-links.
+
+To discover other kinds of documentation:
+- [nix.dev](https://nix.dev/): Documentation entry-point including tutorials and guides for the Nix package manager and the Nix expression language
```

Comment:
```suggestion
- [nix.dev](https://nix.dev/): Tutorials and guides for learning about and achieving goals with Nix
```

Reply:
Why not build on the upstream tagline: 

```suggestion
- [nix.dev](https://nix.dev/): Tutorials and guides for getting things done with Nix
```

Reply:
Applied @fricklerhandwerk's final suggestion

Reply:
Not sure if _learning_ is _getting things done_, but close enough.